frontend/src/index.css

```diff
@@ -5,9 +5,15 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: url("https://cdn.pixabay.com/photo/2014/07/10/13/48/starry-sky-389083_960_720.jpg");
```

Comment:
I wanted to say that this was "stämningsfullt" but had to look up the translation and got a double sided answer from google "This is atmospheric" 😅